### ISSUE TITLE
monorepo_build.sh: automate getting the list of all the remote

### ISSUE THREAD

**Issue by dandv** [shopsys/shopsys#562]
I was reading through https://github.com/shopsys/monorepo-tools#3-building-the-monorepo:

> Then you can build your monorepo using monorepo_build.sh. Just list the names of all your previously added remotes as arguments.

That's a ton of copy/pasting. Would it be possible for the script to parse the output of `git remote -v` and process all the remotes itself?

**Comment by PetrHeinz** [comment]
Hi @dandv and thanks for the idea.

It would be pretty cool to just call the `monorepo_build.sh` and be done with it. You can name the remotes almost any way you want, you you wouldn't even need the possibility to change the target directory (even though there are limitations - for example, remote name cannot contain whitespace or a dot as a first character or immediately after a slash).

Maybe it would make sense to allow calling the `monorepo_build.sh` without any parameters, which would just take all remotes (with the exception of `origin`) and take their names as target directories. Calling `monorepo_split.sh` without parameters would then split all the remotes.

Basically, without parameters both scripts would then behave just like calling `git remote | grep -v ^origin$ | xargs monorepo_build.sh` (or `monorepo_split.sh`).

Calling the scripts with parameters should, in my opinion, behave exactly like it does now for BC and more complex use cases (eg. requiring having a space in the directory name ).

This would make a basic use case much quicker and easier to write. What do you think? Is it worth the additional complexity? Did you have a different solution in mind?
